project-reports/2024/2024-annual-Hyperledger-AnonCreds.md

```diff
@@ -0,0 +1,84 @@
+# 2024 Annual Review Hyperledger AnonCreds
+
+---
+layout: default
+title: 2024 Annual Review Hyperledger AnonCreds
+parent: 2024
+grand_parent: Project Updates
+---
+
+## Project Health
+
+_Guidance: Include a link to your project’s [LFX Insights page](https://insights-v2.lfx.linuxfoundation.org/projects). We will be looking for signs of consistent or increasing contribution activity. Please feel free to add commentary to add color to the numbers and graphs we will see on Insights._
```

Comment:
You can remove the instructions from the final report :-)

Reply:
I debated that, and thought they were useful to have for the casual reader, but I can remove them.

Reply:
Removed.